packages/assets-controllers/src/AssetsContractController.test.ts

```diff
@@ -112,6 +122,9 @@ function mockNetworkWithDefaultChainId({
   });
 }
 
+// eslint-disable-next-line jest/no-export
+export { setupControllers, mockNetworkWithDefaultChainId };
```

Comment:
[nit] If we're going to export `setupControllers` for use in the `AssetsContractControllerWithNetworkClientId` tests maybe we should rename it to `setupAssetContractControllers` so people don't accidentally try to use it for other testing purposes or try to make it more general purpose.

Reply:
great point